TOMP-API.yaml

```diff
@@ -1370,36 +1370,61 @@ components:
           items:
             $ref: '#/components/schemas/card'
         requirements:
-          description: Requirements the users has ((dis)abilities, share [TRUE|FALSE], preferences [TBD]). See also 'https://github.com/efel85/TOMP-API/blob/master/documents/Woordenboek%20Reizigerskenmerken%20CROW%20Eindversie%208%20mei%202019.pdf' [https://github.com/efel85/TOMP-API/issues/17 and https://github.com/efel85/TOMP-API/issues/27]
-          type: array
-          items:
-            $ref: '#/components/schemas/key-value'
- 
+          $ref: '#/components/schemas/requirements'
+
     license:
       type: object
+      description: driver or usage license for a specific user. Contains the number and the asset-type you're allowed to operate (e.g. driver license for CAR)
       properties:
+        number:
+          type: string
+          example: 1287948792
+        assetType:
+          $ref: '#/components/schemas/assetClass'
+        licenseCode:
+          description: in most countries a driver license has also a code. As TO you can exactly verify, based on this code if the license allows to operate it's assets, if the asset-type too generic.
+          example: D4
+          type: string
         country:
-          $ref: '#/components/schemas/country' 
-        asset-type:
-          $ref: '#/components/schemas/asset-class' 
+          $ref: '#/components/schemas/country'
+        validUntil:
+          $ref: '#/components/schemas/date'
 
     card:
       type: object
+      description: card object. Only provide the cards that are requested. The asset type property is only allowed for the DISCOUNT card in combination with certain card-acceptors.
+      required:
+        - cardType
+        - cardNumber
+        - validUntil
       properties:
-        card-type:
+        cardType:
+          type: string
+          enum: [ID, DISCOUNT, TRAVEL, BANK, CREDIT, PASSPORT, OTHER]
+        cardSubType:
+          description: mandatory in case of OTHER. Can be used in bilateral agreements.
+          type: string
+        cardDescription:
+          description: description of the card
           type: string
-        card-acceptors:
-          description: references to maas-ids of accepting parties
+        cardAcceptors:
+          description: references to maas-ids of accepting parties. Only if applicable (DISCOUNT).
           type: array
           items:
             type: string
-        card-number:
+        cardNumber:
+          description: number of the card, like ID number, credit card or bank account number
           type: string
+        cardAdditionalNumber:
+          description: additional number, like CVC code or IBAN code
+          type: string
+        validUntil:
+          $ref: '#/components/schemas/date'
         country:
-          $ref: '#/components/schemas/country' 
-        asset-type:
-          $ref: '#/components/schemas/asset-class' 
- 
+          $ref: '#/components/schemas/country'
+        assetType:
```

Comment:
This confuses me. We have a typeOfAsset object but the property assetType refers to an assetClass...

Can this property be named assetClass?

Reply:
yes. 0.5 is a breaking version, please do so.